apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -424,16 +425,25 @@ function initializeBlocklyWrapper(blocklyInstance) {
         },
         null
       );
+
+      // Core Blockly requires a container div to be LTR, regardless of page direction.
+      container.setAttribute('dir', 'LTR');
+      container.style.display = 'inline-block';
       container.appendChild(svg);
       svg.appendChild(workspace.createDom());
       Blockly.Xml.domToBlockSpace(workspace, xml);
 
-      // Loop through all the child blocks and remove transform
+      // Loop through all the parent blocks and remove vertical translation value
+      // This makes the output more condensed and readable, while preserving
+      // horizontal translation values for RTL rendering.
       const blocksInWorkspace = workspace.getAllBlocks();
       blocksInWorkspace
         .filter(block => block.getParent() === null)
         .forEach(block => {
-          block.svgGroup_.removeAttribute('transform');
+          const svgTransformList = block.svgGroup_.transform.baseVal;
```

Comment:
You'll probably have to deal with a lot of these kinds of "dunno what's going on" questions from me on Blockly PRs for a while :). What exactly is going on here? Is this just doing the same thing it did previously, just using up to date Blockly?

Reply:
Got it, thanks Mike!